src/Components/Home.js

```diff
@@ -1,14 +1,63 @@
 import "../CSS/Home.css";
+import React, {useEffect, useState} from 'react';
 import SWECCmap from "../Data/img/location.png";
 import UWlogo from "../Data/img/UW_logo.png";
 import { InstagramEmbed } from "react-social-media-embed";
 
+import img1 from "../Data/img/backgroundImg/1.jpg"
+import img2 from "../Data/img/backgroundImg/2.jpg"
+import img3 from "../Data/img/backgroundImg/3.jpg"
+import img4 from "../Data/img/backgroundImg/4.jpg"
+import img5 from "../Data/img/backgroundImg/5.jpg"
+import img6 from "../Data/img/backgroundImg/6.jpg"
+import img7 from "../Data/img/backgroundImg/7.jpg"
+
 function HomePage() {
-  const meetingInfo = (
-    <div className="meetingsSection">
-      <div className="meetingsInfoSection">
-        <h2 className="sweccMeetings">SWECC Meetings</h2>
-        <p className="meetingInfo">Held in the EAC, these meeting feature topics in a vast number of areas, including professional development, resume building, and mentor circles</p>
+
+    function Carousel() {
+        const [currentIndex, setCurrentIndex] = useState(0);
+        const backGroundImgs = [img1, img2, img3, img4, img5, img6, img7];
+
+        useEffect(() => {
+            const intervalId = setInterval(nextImg, 6000);
+
+            return () => clearInterval(intervalId);
+        }, [backGroundImgs]);
+
+        const nextImg = () => {
+            setCurrentIndex((prevIndex) => (prevIndex === backGroundImgs.length - 1 ? 0 : prevIndex + 1));
+        };
+
+        const prevImg = () => {
+            setCurrentIndex((prevIndex) => (prevIndex === 0 ? backGroundImgs.length - 1 : prevIndex - 1));
+        };
+
+        const divStyle = {
+            backgroundImage: `linear-gradient(rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.5)), url(${backGroundImgs[currentIndex]})`,
+            backgroundSize: 'cover',
+            backgroundPosition: 'center',
+            padding: '20px',
+            transition: 'opacity 3s ease'
+        };
+
+        return (
+            <div className="clubTitleSection" style={divStyle}>
+                <div className="buttons-container">
+                    <button onClick={prevImg} className="prevImgButton">&lt;</button>
```

Comment:
on arrow button clicks, add css transitions to make this smoother. Same for when the 6000 ms are up and the photo in the carousel changes. Any time the photo changes, add a smooth transition with css